packages/bloc/lib/src/emitter.dart

```diff
@@ -76,7 +76,7 @@ class _Emitter<State> implements Emitter<State> {
     Stream<T> stream, {
     required void Function(T data) onData,
     void Function(Object error, StackTrace stackTrace)? onError,
-  }) async {
+  }) {
```

Comment:
Thanks for catching this! Wondering if it would actually cause any issues 🤔

Reply:
I tried to think about it a bit but I have no idea 😂

Reply:
Haha yeah same here